indico/modules/events/persons/controllers.py

```diff
@@ -212,3 +224,18 @@ def _process(self):
         flash(ngettext('Submission rights have been revoked from one principal',
                        'Submission rights have been revoked from {} principals', count).format(count))
         return redirect(url_for('.person_list', self.event_new))
+
+
+class RHEditEventPerson(RHPersonsBase):
+    def _checkParams(self, params):
+        RHPersonsBase._checkParams(self, params)
+        self.person = EventPerson.query.with_parent(self.event_new).filter_by(id=request.view_args['person_id']).one()
+
+    def _process(self):
+        form = EventPersonForm(obj=self.person)
```

Comment:
`obj=FormDefaults(self.person, skip_attrs={'title'}, title=self.person._title)`; otherwise the title is not populated